Rakefile

```
@@ -73,6 +74,10 @@ db_namespace = namespace :db do
   private
 
   def db_path
-    ActiveRecord::Base.configurations['development']['database']
+    if ActiveRecord.version.to_s.to_f >= 6.1
```

```suggestion
    if ActiveRecord.version.to_s >= "6.1"
```
let's compare strings here 👍